www-plugins/passff/passff-1.3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit mozextension
+
+DESCRIPTION="zx2c4 pass manager extension for Firefox"
+HOMEPAGE="https://github.com/passff/passff"
+SRC_URI="https://addons.mozilla.org/firefox/downloads/file/914592/passff-1.3-an+fx-linux.xpi"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+DEPEND="|| ( www-client/firefox www-client/firefox-bin )"
+RDEPEND="
+	${DEPEND}
+	www-plugins/passff-host
+"
+
+S="${WORKDIR}"
+
+src_unpack() {
+	xpi_unpack "passff-1.3-an+fx-linux.xpi"
+}
+
+src_install() {
+	# taken from app-crypt/eid-mw
+	declare MOZILLA_FIVE_HOME
+	if has_version '>=www-client/firefox-3.6.20'; then
```

Comment:
This looks like automagic. It's bad because if someone installs the other firefox(-bin) variant after building this package, the file is going to be missing. Either use USE flags, or unconditionally install to both (if possible).